### PR TITLE
LPS-112983 Remove the `.js` prefix in stylesheets

### PR DESCRIPTION
**Pull request by julien**
The original discussion was started in #249, and more specifically [here](https://github.com/wincent/liferay-portal/pull/249#issuecomment-624717901)

The `js` CSS class is added by `Liferay.BrowserSelectors` [here](https://github.com/liferay/liferay-portal/blob/17bf6add3fcda2c46093a32d61442c9e97669bfa/modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/browser_selectors.js#L237) and was used in  a couple of places, some usages were in `test/` directories and those usages haven't been updated because it didn't seem necessary - see this [comment](https://github.com/wincent/liferay-portal/pull/249#issuecomment-624734400)

### CODE REVIEW

**Comment by julien** [comment]
ci:test:sf

**Comment by wincent** [comment]
For traceability, would you mind updating the description to point back to the original discussion in https://github.com/wincent/liferay-portal/pull/249 @julien?

**Comment by julien** [comment]
Sure, I'll do that

File: modules/apps/frontend-css/frontend-css-web/src/main/resources/META-INF/resources/portal/_panel_component.scss [hunk 1]
```diff
@@ -89,15 +89,13 @@
 	border-top: 1px solid transparent;
 }
 
-.js {
-	.lfr-floating-container {
-		position: absolute;
-	}
+.lfr-floating-container {
+	position: absolute;
+}
```
Comment (wincent) [inline modules/apps/frontend-css/frontend-css-web/src/main/resources/META-INF/resources/portal/_panel_component.scss:94]: So note that you now have this class declared twice; you should fold this into the other `.lfr-floating-container` down below.

**Comment by wincent** [comment]
Continuing the chain of PRs eager to get out into the wide open world (previously: [#273](https://github.com/wincent/liferay-portal/pull/273#issuecomment-631390518) and [#267](https://github.com/wincent/liferay-portal/pull/267#issuecomment-631594328)), I think we may as well send this one as well.

**Comment by wincent** [comment]
ci:forward

**Comment by wincent** [comment]
"Unique" `Please fix semantic versioning` failure looks spurious to me so I'm going to rebase and retry.

**Comment by wincent** [comment]
ci:forward

**Pull request closed without merging**